### DISCUSSION TITLE
Welcome to libbpf-rs Discussions!

### DISCUSSION THREAD

**Discussion by danielocfb** [libbpf/libbpf-rs#810] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by mabreyb32** [comment]
I am getting the following error when trying to compile my main.rs file

"error[E0599]: no method named `load` found for struct `OpenProgramImpl<'_, Mut>` in the current scope
  --> src/main.rs:15:49
   |
15 |             let loaded_prog: OpenProgram = prog.load().unwrap();
   |                                                 ^^^^ method not found in `OpenProgramImpl<'_, Mut>`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `app_control_agent` (bin "app_control_agent") due to 1 previous error"

Has anybody ran into this issue?

**Reply by danielocfb** [reply]
There is no method `load` on an `OpenProgram`, it's a property of skeletons. https://docs.rs/libbpf-rs/0.24.6/libbpf_rs/skel/trait.OpenSkel.html

**Comment by mabreyb32** [comment]
Thanks that helped however, as I tried to compile I got all the way to the end and then was hit wit the following error:
`error[E0463]: can't find crate for `bindgen`
 --> build.rs:1:1
  |
1 | extern crate bindgen;
  | ^^^^^^^^^^^^^^^^^^^^^ can't find crate

For more information about this error, try `rustc --explain E0463`.
error: could not compile `app_control_agent` (build script) due to 1 previous error
warning: build failed, waiting for other jobs to finish..."`
